src/core/server/logging/layouts/pattern_layout.test.ts

```diff
@@ -133,7 +135,7 @@ test('`format()` correctly formats record with meta data.', () => {
       pid: 5355,
       meta: {},
     })
-  ).toBe('[2012-02-01T14:30:22.011Z][DEBUG][context-meta]{} message-meta');
+  ).toBe('[2012-02-01T09:30:22.011-05:00][DEBUG][context-meta]{} message-meta');
```

Comment:
Same note here - to ensure these tests run predictably we should probably be mocking the timezone (or explicitly providing it in the tests `%date{ISO8601_TZ}{MST}`). Unless we are mocking the jest global timezone somewhere that I'm not aware of.

Reply:
@lukeelmers We have a [global timezone mock](https://github.com/elastic/kibana/blob/master/packages/kbn-test/src/jest/setup/mocks.js#L26-L35) in setup to `America/New_York`.